essential-projects/messagebus
# 🔥 remove unused import

## What did you change?

I removed an unused import

## How can others test the changes?

You can check if nothing breaks. This is a very safe PR, as it just removes an unused import

## PR-Checklist

Please check the boxes in this list after submitting your PR:
